### ISSUE TITLE
Kill rate and team performance tabs to show resolved (custom status) + complete show as complete

### ISSUE THREAD

**Issue by heints** [campai-bv/autotask-dashboards-os#82]
Hello,

We have a challenge that we are using two statuses for complete:

a) Resolved - where we believed it is resolved (equals status "19" in our case)
b) Complete - Autotask standard status (equals status "5")

Is it possible to to make an option to choose inside the modules, what statuses to involve in result display? EG. Kill rate complete means resolved + complete...

Or is it possible to modify the widgets in such a matter that complete = status 5+18 ?

I believe also others will use intermediate statuses for different things :)

Thank You and BR,

Hendrik

**Comment by heints** [comment]
Addition: we would also need somehow to deduct this status (resolved eg. status "19") from the open tickets ;) in the widgets as well!

**Comment by heints** [comment]
Hei! Any thoughts on this. I know it is more like a feature requests to get some option to classify (in dictionary style) the ticketing system statuses and milestones, but I think others will also benefit from this.

Im willing to discuss also getting this coding as a service for sensible fee ( should be upgrade proof)

Hendrik

**Comment by coencoppens** [comment]
@heints Hi Hendrik,

This feature is something we've heard from others as well. In fact, we've even had a few discussions about this internally :-)

We still have a status "resolved" too. This is used by the Servicedesk to see which tickets are taken care off but can't be completed yet: the customer has 2 days to submit comments to the ticket. After those 2 days, if no comments are added, it automagically gets completed. That means the ticket effectively isn't completed, it's basically a "Waiting customer but only for 2 days". So it doesn't count as completed.

Long story short, I don't think adding it as a standard feature would make sense. Completed = completed, the rest isn't. But if you want I can check to see how you could make this change as a upgrade-proof extension.

**Comment by mattrbailey** [comment]
we are also after something similar to this as well. Please can you let me know if there are any plans to implement this?
